build.fsx

```diff
@@ -141,26 +141,11 @@ Target "SourceLink" (fun _ ->
 // Build a NuGet package
 
 Target "NuGet" (fun _ ->
-    let fsiVersion = GetPackageVersion "packages" "FSharp.Compiler.Service"
-    let scriptcsVersion = GetPackageVersion "packages" "ScriptCs.Hosting"
-    NuGet (fun p -> 
+    Paket.Pack (fun p ->
         { p with   
-            Authors = authors
-            Project = project
-            Summary = summary
-            Description = description
```

Comment:
I get it now. I must have been too tired to comprehend when I tried this last night.

All the same, I very much like having all the info in one file, or at least separate `fsx` files rather than text files, but that's just my preference.